internal/receiver/databricksreceiver/metadata.yaml

```diff
@@ -85,303 +94,337 @@ metrics:
       value_type: int
     attributes:
       [job_id, task_id]
+
+  databricks.spark.codegenerator.compilationtime.mean:
```

Comment:
What does `mean` stand for?

Reply:
Arithmetic mean -- aka average. It's taken from a spark histogram, but we don't have histogram support in the metrics builder code, and this metric seems not very interesting, so I just extracted the mean value.

Reply:
the histogram bit is interesting. I'd write in the description what you say here in your comment so folks can follow how this is coming together.

Reply:
Good idea, will do.

Reply:
This is done.